demo/kserve/scripts/test/http-call.sh

```diff
@@ -4,24 +4,36 @@ set -o nounset
 set -o errtrace
 # set -x   #Uncomment this to debug script.
 
+# Performs inference using HTTP
+
+PREFIX=""
+INF_PROTO=""
+
 source "$(dirname "$(realpath "$0")")/../env.sh"
 source "$(dirname "$(realpath "$0")")/../utils.sh"
 
 echo
-echo "Wait until runtime is READY"
+echo "Wait until $INF_PROTO runtime is READY"
```

Comment:
I'd also add `set -u` to make sure this fails when `INF_PROTO` is unset (here and in the grpc script).

How about renaming `INF_PROTO` to just `PROTOCOL` and removing the `PREFIX` variable?

Reply:
You are right, I will remove PREFIX and even INF_PROTO can be removed as it is involved in only 2 lines and can be hard coded, especially as the convention is not symmetric: 
for grpc the runtime is named "caikit-tgis-runtime-grpc" while for http it is named "caikit-tgis-runtime"
so that if we put htpp/grpc in variable "PROTOCOL"  we can not even write
caikit-tgis-runtime-{$PROTOCOL}.  I should push new code today

Reply:
@dtrifiro That's it. I pushed improvements to both grpc-call and http-call.  Please review

Reply:
@dtrifiro  I also added your suggested "set -u" in both scripts to prevent any non initialized env. variable